### DISCUSSION TITLE
Show token icon on "empty" option?

### DISCUSSION THREAD

**Discussion by Luebbi81** [JDCalvert/FVTT-PF2e-Ranged-Combat#94] [Ideas]
This mod is great and helps me keep track of my dual-wielding gunslinger player, so thanks!

Since he's dual wielding, he always has two "loaded" status effects on his token. He's playing a small character and we have increased status icon size due to my sight issues, which means his token is halfway covered with loaded icons even while out of combat and exploring.

It would be great if there could be an option to reverse this - instead of showing a loaded status, show an empty status once a gun has been fired. This would be especially useful for characters wielding multiple ranged weapons. 

I don't know how feasible this would be, but thanks if you're even considering it :)

**Comment by JDCalvert** [comment]
Hi there, thanks for taking the time to make this suggestion! Unfortunately I don't think exactly what you're suggesting would be possible as the "effects" created by the module hold quite a lot of information that otherwise we wouldn't have access to. However, I could add an option so the effects don't show on the token. They would still be visible at the top right of the canvas when the character is selected. Would that be useful to you?

**Reply by Luebbi81** [reply]
Hi, that would be sweet! I was thinking that "reversing" the effect would probably not be viable, but your option would keep the token clean while giving the player the info he needs!

**Reply by JDCalvert** [reply]
Hi, this has been added in the latest release (3.3.0)! It's a client setting, so either your player will have to go into settings and set it, or you can use something like Force Client Settings to set it for everyone.
